src/main/resources/data/spectrum/worldgen/configured_feature/bristle_sprouts.json

```diff
@@ -28,11 +28,6 @@
           "type": "minecraft:block_predicate_filter",
           "predicate": {
             "type": "minecraft:matching_blocks",
-            "offset": [
-              0,
-              -1,
-              0
-            ],
             "blocks": "spectrum:sawblade_grass"
```

Comment:
Wait, wouldn't the predicate search for air and sawblade grass at the same pos now? o.O

Reply:
It still generates successfully, due to mojank? But yeah that is a good point, the sawblade grass check might not be doing anything here to begin with, seeing how it still generates despite being seemingly impossible if this was doing something

Reply:
Wait never mind I must've been very tired when i was testing it, yeah this just makes it never generate at all, I was confusing it with sawblade holly bushes, yeah this needs to be fixed